README.md

```diff
@@ -1,2 +1,141 @@
-# react-avatar-creator
-A react wrapper for the Ready Player Me Avatar Creator
+# Ready Player Me - React Avatar Creator
+
+**Ready Player Me - React Avatar Creator** is a set of components and helper methods to help implementing the Ready Player Me Avatar Creator into React projects.
+
+## Installation
+Ready Player Me React Avatar Creator is available as a [npm package](https://www.npmjs.com/package/@readyplayerme/react-avatar-creator).
+
+```bash
+npm i @readyplayerme/react-avatar-creator
+```
+
+## Usage
+
+```tsx
+import { AvatarCreator } from '@readyplayerme/react-avatar-creator';
+
+export default function App() {
+  return (<AvatarCreator subdomain="demo"/>);
+}
+```
+
+https://user-images.githubusercontent.com/3163281/235168912-a9dacd91-af3a-4b35-81c3-b025e12e333a.mp4
+
+---
+
+# Components
+
+## AvatarCreator
+
+AvatarCreator component helps you load Ready Player Me in an iframe where you can edit your avatar and receive your avatar URL as a post message once your avatar is exported.
+
+### Parameters
+
+**subdomain** *[required]*: string 
+- Your Ready Player Me subdomain. You can get one from [Ready Player Me Studio](https://studio.readyplayer.me/).
+
+**className** *[optional]*: string
+- The css styles to apply to the iframe.
+
+**config** *[optional]*: AvatarCreatorConfig
+- Editor Configuration is where you can set url properties of Ready Player Me editor. Read more about these options in [Ready Player Me documentations](https://docs.readyplayer.me/ready-player-me/integration-guides/web-and-native-integration/avatar-creator-integration#configuration-1).
+
+**onAvatarExported** *[optional]*: (event: AvatarExportedEvent) => void
+- Callback function that is called when avatar is exported.
+
+**onUserSet** *[optional]*: (event: UserSetEvent) => void
+- Callback function that is called when user id is set.
+
+**onAssetUnlocked** *[optional]*: (event: AssetUnlockedEvent) => void
+- Callback function that is called when an asset is unlocked.
+
+**onUserAuthorized** *[optional]*: (event: UserAuthorizedEvent) => void
+- Callback function that is called when the user is authorized.
+
+### Example
+
+```tsx
+import { AvatarExportedEvent, UserSetEvent } from "@readyplayerme/react-avatar-creator/events";
+import { AvatarCreator, AvatarCreatorConfig } from "@readyplayerme/react-avatar-creator";
+
+const config: AvatarCreatorConfig = {
+    clearCache: true;
+    bodyType: 'halfbody';
```

Comment:
I thought I had put this in the comments already but... 
A super minor thing, perhaps these example values should be set to the most commonly used values. 
EG
```
bodyType: 'fullbody';
language: 'en';
```
Just thinking that if people copy and paste blindly they might wonder why their avatar or language doesn't match what they expect.

Reply:
Makes total sense to me to do this - will include 👍

Reply:
Done.